libfreerdp/utils/test/TestFuzzReadPassphrase.c

```diff
@@ -0,0 +1,6 @@
+#include <freerdp/utils/passphrase.h>
+
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  freerdp_passphrase_read(NULL, (char*)Data, Size, 0);
```

Comment:
not a good idea, this function will wait for terminal input

Reply:
I thought `0` disables STDIN input. Isn't it?

Reply:
yes, but it then reads from tty ;)